frontend/model/getters.js

```diff
@@ -209,16 +220,16 @@ const getters: { [x: string]: (state: Object, getters: { [x: string]: any }) =>
       })
     }
   },
-  ourPaymentsSentInPeriod (state, getters) {
-    return (period) => {
+  ourPaymentsSentInPeriodForGroup (state, getters) {
+    return (state, period) => {
```

Comment:
`getters.groupPeriodPayments` on line below

Reply:
It looks like I missed quite a few of these. I'll review all of the `ForGroup` getters as well.